### ISSUE TITLE
Unable to open display

### ISSUE THREAD

**Issue by GodloveD** [LizardByte/Sunshine#486]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your issue described in the documentation?

- [X] I have read the documentation

### Describe the Bug

I'm trying to install Sunshine on a GPU equipped cloud node.  I'm using the Vultr platform and have selected one of their NVIDIA A40 nodes.  I know that these nodes can support remote graphics hardware acceleration because I've previously used the same instance type to install VirtualGL and tested it at 60fps with the Unigine Valley benchmark. That works fine. 

However, I can't get out of the starting gate with Sunshine.  I've tried installing it on an Ubuntu 20.04 instance via the .deb package and also by building from source and also via the AppImage package as described in the documentation.  In all cases I get the configuration done (also as described there) reboot the node, run the Sunshine command and I get the following: 
```
# sunshine
[2022:11:08:23:43:33]: Error: Failed to create session: Unable to open display
[2022:11:08:23:43:33]: Error: Couldn't expose some/all drm planes for card: /dev/dri/card1
```
If I do an `export DISPLAY=:0` I get the following:
```
# sunshine
[2022:11:08:23:54:50]: Error: Failed to create session: Unable to open display
[2022:11:08:23:54:50]: Error: Couldn't expose some/all drm planes for card: /dev/dri/card1
[2022:11:08:23:54:50]: Info: Detecting connected monitors
```
Nothing more.  

I've seen several suggestions in the issues that folks use the flatpak package, and I've tried that as well.  But that seems to result in a architecture mismatch.  Weird because this instance is x86_64.
```
# flatpak run dev.lizardbyte.sunshine
bwrap: execvp ldconfig: Exec format error
error: ldconfig failed, exit status 256
```
I've seen mention in the issues about logs being saved somewhere, but I'm unable to locate any sunshine related logs on the system.  So I'm kindof stuck.  How can I proceed to debug this issue?  Is it just that Sunshine won't run on a headless server like this?  As I mentioned before server side graphics are totally possible on a system like this using VirtualGL.  

### Expected Behavior

I expect to at least be able to get the web server running so that I can connect and start configuring.  

### Host Operating System

Linux

### Operating System Version

I've tried on both Ubuntu 20.04 and 22.04.  (Also tried on Fedora but got unhelpful seg fault message)

### Architecture

64 bit

### Sunshine commit or version

I've tried with 0.15.0 packages and commit e1b112c building from source

### Package

Linux - deb, AppImage, built from source, and Flatpak

### GPU Type

Nvidia

### GPU Model

a40

### GPU Driver/Mesa Version

510.85.02

### Capture Method (Linux Only)

unknown

### Relevant log output

I would love to do this if I could find logs.

**Comment by ReenigneArcher** [comment]
Logs is generally just the terminal output after executing `sunshine`, although if running as a service they're stored on the system.

For most types of encoding you'll need some type of monitor whether it be a physical monitor or a monitor dongle. If I'm wrong about that I'm sure someone will correct me.

I know you can stream on a headless server using X11 with software encoding though. Try to run `sudo setcap -r $(readlink -f $(which sunshine))` followed by a system restart.

The command is in our docs as well. https://docs.lizardbyte.dev/projects/sunshine/en/latest/about/usage.html#linux

**Comment by GodloveD** [comment]
Thanks for the quick reply.  I should have specified that I did execute the command `sudo setcap -r $(readlink -f $(which sunshine))` followed by a restart and still got the same problem.  However, I'm not actually sure if I had X11 installed and the xserver running when I did that so it's worth double checking.  

Something you said interested me and I'd like to request clarification.  
>I know you can stream on a headless server using X11 with software encoding though. 

When you say "software encoding" you mean that the remote GPU won't be used right?  I'm really looking to accomplish hardware encoding on a headless remote server.  Is that possible with moonlight or am I barking up the wrong tree here?  

Thanks for your responsiveness!  :smiley_cat:
